searx/webapp.py

```diff
@@ -254,6 +255,17 @@ def code_highlighter(codelines, language=None):
     return html_code
 
 
+def spellcheck(query: str, user_language: str):
+    spell = SpellChecker(language=user_language)
+    spellcheck_list = query.split(' ')
+    spellcheck_list = [x for x in spellcheck_list if x != '']
+    has_misspelled_words = bool(spell.unknown(spellcheck_list))
```

Comment:
```suggestion
    has_misspelled_words = any(spell.unknown(spellcheck_list))
```
should do the same I think and is more readable